packages/devtools_app/lib/src/shared/dialogs.dart

```diff
@@ -115,17 +115,17 @@ class DialogApplyButton extends StatelessWidget {
 class DialogTextButton extends StatelessWidget {
   const DialogTextButton({this.onPressed, this.child});
 
-  final VoidCallback onPressed;
+  final VoidCallback? onPressed;
 
-  final Widget child;
+  final Widget? child;
```

Comment:
make this non-nullable and required if we are going to force unwrap it in the build method anyway. This is also consistent with TextButton

Reply:
done